### ISSUE TITLE
How to train OSNet from scratch

### ISSUE THREAD

**Issue by DeepAlchemist** [KaiyangZhou/deep-person-reid#217]
Hi, thanks for your code. I try to train OSNet from scratch according to the implementation details mentioned in your paper, and get the following results,

mAP: 74.8%
CMC curve
Rank-1  : 89.9%
Rank-5  : 96.3%
Rank-10 : 97.4%
Rank-20 : 98.3%
Checkpoint saved to "../log/osnet_x1_0-mk-softmax-scratch/model.pth.tar-350"
Elapsed 5:38:53

The training script is,
python main.py \
-j 4 \
--app image \
--loss softmax \
--label-smooth \
-s market1501 \
-a osnet_x1_0 \
--height 256 \
--width 128 \
--optim sgd \
--lr 0.065 \
--max-epoch 350 \
--stepsize 150 225 300 \
--lr-scheduler multi_step \
--batch-size 64 \
--transforms random_flip \
--save-dir ./log/osnet_x1_0-mk-softmax-scratch \
--start-eval 1 \
--eval-freq 10 \
--gpu-devices 4

Due to I missed random erasing?

**Comment by KaiyangZhou** [comment]
add random_crop and random_erase
